app/survey_config/census_config.py

```diff
@@ -71,6 +35,54 @@ def get_data_layer(self, tx_id: Optional[str] = None) -> list[dict]:
 
         return data_layer
 
+    def get_footer_links(self, cookie_has_theme: bool) -> list[dict]:
+        links = [
+            Link(
+                lazy_gettext("Help"),
+                f"{EN_BASE_URL}/help/how-to-answer-questions/online-questions-help/",
+            ).__dict__
+        ]
+
+        if cookie_has_theme:
+            links.append(
+                Link(lazy_gettext("Contact us"), f"{EN_BASE_URL}/contact-us/").__dict__
+            )
+
+        links.extend(
+            [
+                Link(
+                    lazy_gettext("Languages"),
+                    f"{EN_BASE_URL}/help/languages-and-accessibility/languages/",
+                ).__dict__,
+                Link(
+                    lazy_gettext("BSL and audio videos"),
+                    f"{EN_BASE_URL}/help/languages-and-accessibility/accessibility/accessible-videos-with-bsl/",
+                ).__dict__,
+            ]
+        )
+
+        return links
+
+    def get_footer_legal_links(self, cookie_has_theme: bool) -> Optional[list[dict]]:
+        if cookie_has_theme:
+            return [
+                Link(lazy_gettext("Cookies"), f"{EN_BASE_URL}/cookies/").__dict__,
+                Link(
+                    lazy_gettext("Accessibility statement"),
+                    f"{EN_BASE_URL}/accessibility-statement/",
+                ).__dict__,
+                Link(
+                    lazy_gettext("Privacy and data protection"),
+                    f"{EN_BASE_URL}/privacy-and-data-protection/",
+                ).__dict__,
+                Link(
+                    lazy_gettext("Terms and conditions"),
+                    f"{EN_BASE_URL}/terms-and-conditions/",
+                ).__dict__,
+            ]
+
+        return []
```

Comment:
Minor, but the types for these methods among the classes should be consistent. Either all should be Optional list of dict, or just list of dict. If you are returning an empty list, then you don't need the optional bit. Alternately and my preference is unless you do a list operation on the values that get returned (it doesn't look like it), I would return None.

Reply:
My understanding was empty list is evaluated as `None` but yeah, I will return `None` explicitly and make it Optional where it applies.

Reply:
Ya, it does, but if we don't do any list operation on it, then we should just return None.